README.md

```diff
@@ -22,7 +22,7 @@ Go forth and make awesome! And, once you've built something great, [send us feat
 
 ## Installation ##
 
-This assumes you have a PHP environment with the required Redis library and a working Redis server (e.g. Pantheon).
+This assumes you have a PHP environment with the [required PhpRedis library](https://github.com/phpredis/phpredis) and a working Redis server (e.g. Pantheon).
```

Comment:
Could we have "PhpRedic extension" here? The repo says "A PHP extension for Redis".